site/app/templates/polls/ViewPoll.twig

```diff
@@ -168,8 +168,10 @@
             const [correctColor, defaultColor] =
             [window.getComputedStyle(document.body).getPropertyValue('--histogram-correct'),
                 window.getComputedStyle(document.body).getPropertyValue('--histogram-default')];
+            let responseText;
             {% for option in poll.getOptions() %}
-                data[0].x.push("{{ option.getResponse()|replace({"\n": " ", "\r": " ", "\t": " "}) }}");
+                responseText = "{{ option.getResponse()|replace({"\n": " ", "\r": " ", "\t": " "}) }}";
+                data[0].x.push(responseText.replace(/(\d) /g, "$1\u00A0"));
```

Comment:
This uses a RegEx on the client side instead of changing all spaces to no-break spaces with Twig's replace function, because I'm not sure if other behavior will be affected